hana3d/src/ui/callbacks/asset_bar.py

```diff
@@ -32,6 +32,7 @@ def draw_tooltip(   # noqa: WPS211
     revision='',
     img=None,
     gravatar=None,
+    sku=None,
```

Comment:
Aqui seria melhor se fosse um `{}` assim impede um possível erro de: `None has no attribute keys`

Reply:
sku={} gera os erros: 
hana3d/src/ui/callbacks/asset_bar.py:35:9: B006 Do not use mutable data structures for argument defaults.  They are created during function definition time. All calls to the function reuse this one instance of that data structure, persisting changes between them.
hana3d/src/ui/callbacks/asset_bar.py:35:9: WPS404 Found complex default value